src/common/Database/DatabaseWorkerPool.cpp

```diff
@@ -99,6 +95,82 @@ void DatabaseWorkerPool<T>::Close()
     sLog->outSQLDriver("All connections on DatabasePool '%s' closed.", GetDatabaseName());
 }
 
+template <class T>
+uint32 DatabaseWorkerPool<T>::OpenConnections(InternalIndex type, uint8 numConnections)
+{
+    _connections[type].resize(numConnections);
+    for (uint8 i = 0; i < numConnections; ++i)
+    {
+        T* t;
+
+        if (type == IDX_ASYNC)
+            t = new T(_queue, *_connectionInfo);
+        else if (type == IDX_SYNCH)
+            t = new T(*_connectionInfo);
+        else
+            ABORT();
```

Comment:
reason to abort? also include brackets please

Reply:
`ABORT("some reason");`

Reply:
`ABORT_MSG` - not implement in core now